appveyor.yml

```diff
@@ -33,3 +33,10 @@ deploy:
   artifact: /artifacts\/packages\/.+\.nupkg/
   on:
     appveyor_repo_tag: true
+- provider: NuGet
+  api_key:
+    secure: I6vvPtMCX9hyycxE5r/STvJtaMT/dswZq9yDOKDQoTCh+WNOJdw4KduUV3k4GnBw
```

Comment:
Are you exposing the secure ID? This is dangerous, there is key management in GitHub Action.